tests/systemd/services_tests.py

```diff
@@ -128,16 +128,16 @@ def test_node_mainnet_service():
     node_service_test("mainnet")
 
 
-def test_node_limanet_service():
-    node_service_test("limanet")
+def test_node_mumbainet_service():
```

Comment:
There changes seem to be unrelated to the commit
Otherwise LGTM

Reply:
My rebasing skills are far from ideal :see_no_evil: